server/controllers/search/lib/entities_query_builder.js

```diff
@@ -157,3 +161,14 @@ const validatePropertyAndValue = condition => {
     }
   }
 }
+
+function matchLanguageCode (search) {
+  search = search.toLowerCase()
+  const claimTerms = languagesCodesProperties.map(property => `${property}=${search}`)
+  return {
+    terms: {
+      claim: claimTerms,
+      boost: 100,
```

Comment:
Isnt this `boost` an overkill ? Since, for the moment, the client is not searching with both a type `languages` and another type. Even if it does one day, i guess one would also like the exact match of an [entity name with a language string in it](inventaire.io/entity/wd:Q271931).